api/controllers/userController.js

```diff
@@ -0,0 +1,24 @@
+const { userService } = require("../services");
+
+const signIn = async (req, res) => {
+  try {
+    const { accessToken } = req.body;
+    const result = await userService.signIn(accessToken);
+
+    if (result.newUser) {
+      return res
+        .status(200)
+        .json({ message: "SUCCESS_SIGNUP", token: result.token });
+    } else {
+      return res
+        .status(200)
+        .json({ message: "SUCCESS_SIGNIN", token: result.token });
+    }
```

Comment:
코드 작성후 테스트가 된 것인가요?
어디에도 res가 선언 되지 않은 것 같은데? 어떻게 동작한거죠?
그리고 이왕 분리한거 status code로 적절히 사용하면 좋을 것 같네요.
```suggestion
    if (result.newUser) {
      return res
        .status(201)
        .json({ message: "SUCCESS_SIGNUP", token: result.token });
    } else {
      return res
        .status(200)
        .json({ message: "SUCCESS_SIGNIN", token: result.token });
    }
```